modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -46,25 +46,24 @@
      * Setting this to a very low value could have an impact on the performance since the number of DB calls will be
```

Comment:
Looks like you have formatted the whole class. This makes hard to identify the change done. can you include only your change? (I assume that the existing class do not have a formatting issue)

Reply:
The existing class already has formatting issues. To address your point, will adding a separate commit for the new property make sense?

Reply:
It seems some of the formatting changes are not required. Are you using the right margin as 120?

Reply:
Reverted back to previous formatting and included the new property with 0450a50